### ISSUE TITLE
When navigating with a screen reader, it can focus on some invisible Items.

### ISSUE THREAD

**Issue by yzen** [mozilla-mobile/mozilla-vpn-client#399]
It does not happen often but you can see the issue with the following STRs:

* Enable screen reader (such as VO)
* Navigate to Select Location screen
* Use `controls + optinos + →` to move screen reader cursor (not keyboard focus) forward

Expected: screen reader cursor moves from back button to select location header to select location list
Actual: screen reader cursor moves from back button to select location header to some invisible text component and only then to the list. See screenshot below.

This probably means that something is hidden visually (like with opacity) but not from screen readers (perhaps `enabled` property can help). Other screen might be suffering from this issue as well, so it is probably an issue in `VPNMenu`.

![](https://user-images.githubusercontent.com/561275/102374447-5abad900-3f8f-11eb-89a6-25987479e5ba.png)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-89)
┆Reporter: Data Sync

**Comment by ValentinaPC** [comment]
Verified this on Mac OS 10.15.7 and also noted, aside the VPN-external options being selected while screen recorder is on, that the cities inside a country cannot be selected via Voice Over.
VPN version used: 2.0 cross-plat production build.
See below screenshots.
![image](https://user-images.githubusercontent.com/15685960/102505276-7656da00-408a-11eb-8d4c-1fc9174b0821.png)
![image](https://user-images.githubusercontent.com/15685960/102505303-7e167e80-408a-11eb-9c32-4a3c6a9893bf.png)

**Comment by data-sync-user** [comment]
➤ Sarah Bird commented:

Accessibility is important and we're working on a big accessibility review. However since the filing of this bug the UI has changed considerably so closing this as stale.
Closed during bug squashing.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Verified this while using 2.15.0 (2.202305121612) Mozilla VPN build on Windows 10.

The issue is no longer reproducing.
